Core/Steps/SubSteps/ReleaseVersionAndMoveIssuesSubStep.cs

```diff
@@ -32,29 +28,26 @@ public class ReleaseVersionAndMoveIssuesSubStep
       IJiraIssueService jiraIssueService,
       IJiraVersionCreator jiraVersionCreator,
       IJiraVersionReleaser jiraVersionReleaser)
+      : base(console, inputReader, jiraIssueService, jiraVersionCreator)
   {
-    _console = console;
-    _inputReader = inputReader;
-    _jiraIssueService = jiraIssueService;
-    _jiraVersionCreator = jiraVersionCreator;
     _jiraVersionReleaser = jiraVersionReleaser;
   }
 
   public void Execute (SemanticVersion currentVersion, SemanticVersion nextVersion, bool squashUnreleased = false, bool movePreReleaseIssues = false)
   {
-    var currentVersionID = CreateVersion(currentVersion);
-    var nextVersionID = CreateVersion(nextVersion);
+    var currentVersionID = CreateVersion(currentVersion.ToString());
+    var nextVersionID = CreateVersion(nextVersion.ToString());
```

Comment:
Maybe we should add an overload in the base class to do this conversion, allowing us to use the `SemanticVersion` without needing to worry how to correctly convert it.